LayoutTests/imported/w3c/web-platform-tests/screen-wake-lock/wakelock-insecure-context.any-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-PASS Wake Lock API is not exposed in an insecure context
+FAIL Wake Lock API is not exposed in an insecure context assert_false: 'WakeLock' must not be exposed expected false got true
```

Comment:
Do we need a radar to adopt Secure Contexts in more places?

Reply:
Weird -- so we declare this is Secure Context in IDL, but it's still exposed in insecure? That's something we should flag as a separate bug...

Reply:
You'll notice that we have a FAIL here but GTK has a PASS. Our implementation is correct. The reason we fail is that we run WPT tests incorrectly on the Cocoa ports. We're still running them over localhost instead of the web-platform.test domain. As a side effect, localhost is treated as a secure context, even when not HTTPS.
The test doesn't expect to be run over localhost, assumes the context shouldn't be secure, but it is for us. This is why we're failing this.

Reply:
And we already have a radar to fix the fact that we should run WPT tests over web-platform.test domain instead of localhost. I even prepared a WIP PR at https://github.com/WebKit/WebKit/pull/4438. However, EWS is failing some loads due to SSL certificate failures and I haven't been able to figure them out yet. I cannot reproduce those failures locally so it doesn't help.